gdal/ogr/ogrsf_frmts/flatgeobuf/flatbuffers/base.h

```diff
@@ -0,0 +1,379 @@
+#ifndef FLATBUFFERS_BASE_H_
```

Comment:
is it a generated file / license ?

Reply:
No it's the official C++ include files from the flatbuffers project, LICENSE file added (Apache license, v2) to the containing dir.